tests/comfy_cli/registry/test_config_parser.py

```
@@ -212,6 +212,207 @@ def test_extract_node_configuration_with_requires_comfyui():
         assert result.project.supported_comfyui_version == "2.0.0"
 
 
+def _write_pyproject(tmp_path, body: str) -> str:
+    """Write a pyproject.toml in tmp_path and return its absolute path as a string."""
+    p = tmp_path / "pyproject.toml"
+    p.write_text(body)
+    return str(p)
+
+
+def test_dynamic_version_resolved_from_double_quoted_literal(tmp_path):
```

Pylint (4.0.5)

[convention] Missing function or method docstring

(C0116)